years_to_hours.rb

```diff
@@ -0,0 +1,59 @@
+# Constants 
+DAYS_PER_LEAP = 366
+DAYS_PER_YEAR = 365
+HOURS_PER_DAY = 24
+MINUTES_PER_HOUR = 60
+SECONDS_PER_MINUTE = 60
+#MINUTES_IN_DECADE = 5259492
+#SECONDS_IN_YEAR = 31556952
+
+def leap(year)
+    if year > 3
+        leap_years = year / 4.0
+        leap_years = leap_years.ceil
```

Comment:
So I think you might actually want to use `floor` here instead of `ceil` because I beliieeeeve year 0 does not count as a leap year.